src/js/util/Validator.js

```diff
@@ -0,0 +1,9 @@
+export const isNumber = (s) => /^[0-9]*$/.test(s);
+export const isUniqueNumbers = (numbers = []) => {
```

Comment:
```suggestion
export const isUniqueNumbers = (numbers) => {
```
default 값이 꼭 필요할까요? 오히려 항상 numbers가 있어야 하는게 맞는 것 같아요!

Reply:
요 default값은 함수의 구동을 위해서라기보다는 자바스크립트에서 최대한 타입을 명시하려고 노력한 부분이에요!
타입스크립트처럼 자동완성 기능이 입력된다면 실수를 줄일 수 있지 않을까 해서요!
말씀하신 대로 이곳에서는 항상 numbers라고 하는 배열이 입력되어 들어오는 것이 맞습니다.

Reply:
실수가 있다면 오류를 발생시켜서 사용자에게 알려야 하지 않을까요?

Reply:
넵 이 부분은 jsdoc으로 보완하였습니다